---
title: "'Finally' dışında bir 'Try' deyimi yer alamaz"
ms.date: 07/20/2015
f1_keywords:
- vbc30382
- bc30382
helpviewer_keywords:
- BC30382
ms.assetid: 0314d8d2-18bc-4bbd-858c-0a18408b52fd
ms.openlocfilehash: 1d5987d6dc3b25088a55b6f0f43a8fb689f2c3e0
ms.sourcegitcommit: 9b552addadfb57fab0b9e7852ed4f1f1b8a42f8e
ms.translationtype: MT
ms.contentlocale: tr-TR
ms.lasthandoff: 04/23/2019
ms.locfileid: "62027530"
---
# <a name="finally-cannot-appear-outside-a-try-statement"></a>'Finally' dışında bir 'Try' deyimi yer alamaz
`Finally` tamamlamak için kullanılan bir `Try...Catch...Finally` engelle; Bu nedenle yalnızca bir kez bloğun sonunda görünebilir. Ya da gereksiz bir sahip `Finally`, veya `Finally` deyimi, karşılık gelen sınırları dışında görünür `Try` blok.  
  
 **Hata Kimliği:** BC30382  
  
## <a name="to-correct-this-error"></a>Bu hatayı düzeltmek için  
  
1. Bulmak ve gereksiz kaldırma `Finally` deyimi.  
  
2. Taşıma `Finally` kodunuzdaki uygun konuma deyimi.  
  
## <a name="see-also"></a>Ayrıca bkz.

- [Try...Catch...Finally Deyimi](../../visual-basic/language-reference/statements/try-catch-finally-statement.md)
